### PR TITLE
Describe a guide to build an app for all platforms in github actions

### PR DESCRIPTION
**Pull request by dennypenta**
# Description

The PR adds a short guide of building an app for all platforms using GitHub actions.

### CODE REVIEW

File: website/docs/guides/crossplatform-build.mdx [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+      - '*'
+
+env:
+  # Necessary for most environments as build failure can occur due to OOM issues
+  NODE_OPTIONS: "--max-old-space-size=4096"
+
+jobs:
+  build:
+    strategy:
+	  # Failure in one platform build won't impact the others
```
Comment (dennypenta) [inline website/docs/guides/crossplatform-build.mdx:29]: don't get it, locally seems fine

Comment (leaanthony) [reply website/docs/guides/crossplatform-build.mdx:29]: Thanks 🙏 Please could you add an entry to the changelog located at `website/src/pages/changelog.mdx`?

File: website/docs/guides/crossplatform-build.mdx [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+
+Below is a comprehensive example that demonstrates building an app upon the creation of a new Git tag and subsequently uploading it to the Actions artifacts:
+
+```yaml
+name: Wails build
+
+on:
+  push:
+    tags:
+	  # Match any new tag
```
Comment (mmghv) [inline website/docs/guides/crossplatform-build.mdx:19]: This is a tab character, replace it with spaces.

**Review by mmghv — CHANGES_REQUESTED** [review]

File: website/docs/guides/crossplatform-build.mdx [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+      - '*'
```
Comment (mmghv) [inline website/docs/guides/crossplatform-build.mdx:20]: Could also be trigger on version tags only :
```
      - "v*.*.*"
```
Or 
```
      - "v*"
```

**Review by leaanthony — APPROVED** [review]

**Comment by leaanthony** [comment]
Thanks @dennypenta 🙏

**Pull request merged**